localstack/services/awslambda/lambda_utils.py

```diff
@@ -171,6 +180,24 @@ def get_main_endpoint_from_container():
     )
 
 
+def get_container_network_for_lambda():
+    global LAMBDA_CONTAINER_NETWORK
+    if not config.LAMBDA_DOCKER_NETWORK and LAMBDA_CONTAINER_NETWORK is None:
```

Comment:
tiny nit: We could use the "Return Early" pattern here to slightly untangle the combined `if` checks:
```
if config.LAMBDA_DOCKER_NETWORK:
    return config.LAMBDA_DOCKER_NETWORK

if LAMBDA_CONTAINER_NETWORK is None:
    ...
return LAMBDA_CONTAINER_NETWORK